torch/_torch_docs.py

```diff
@@ -6656,6 +6656,29 @@ def merge_dicts(*dicts):
     tensor([ -24.8863, -771.4742])
 """.format(**common_args))
 
+add_docstr(torch.positive,
+           r"""
+positive(input) -> Tensor
+
+Returns :attr:`input`.
```

Comment:
Small tweak: "Returns :attr:\`input\`:"

Reply:
Do you mean that "input" should be surrounded by backtick, or intend to write as "Returns *input*:"?

Reply:
I thought a colon would connect this sentence to the mathematical expression, but I think you're right my suggestion was confusing.

What about leaving this sentence as-is, but just removing the math section? The behavior of the function is clear without it

Reply:
I see! Fixed the documentation as you commented. Thanks!